vonk/configuration/logsettings.rst

```diff
@@ -72,98 +73,113 @@ But in this (purposefully incorrect) example the ``Warning`` level on the ``Vonk
 Changing the sink
 -----------------
 Another setting you can adjust is ``WriteTo``. This tells Serilog which sink(s) to log to.
-Serilog provides several sinks, and for Vonk you can use ``Console``, ``ColoredConsole``, ``RollingFile`` and ``Seq``.
+Serilog provides several sinks, and for Vonk you can use ``Console``, ``File``, ``ApplicationInsights`` and ``Seq``. All of which can be wrapped in an ``Async`` sink to avoid blocking Vonk when waiting for the sink to process the log statements.
 
-RollingFile
-^^^^^^^^^^^
-For the ``RollingFile`` sink, you can specify the location of the log files with the ``pathFormat`` argument.
-Please include the ``{Date}``, ``{Hour}`` or ``{HalfHour}`` placeholder, so Serilog can add date and time
-information to your filename.
+Console
+^^^^^^^
 
-*	Navigate to your Vonk working directory.
-*	Open :code:`logsettings.json` in a text editor and find the ``WriteTo`` setting::
+The Console sink will write to your shell.
+
+*	Find the ``WriteTo`` setting::
 
 		"WriteTo": [
-			{ "Name": "ColoredConsole" },
 			{
-				"Name": "RollingFile",
-				"Args": { "pathFormat": "c:/temp/vonk-{Date}.log" }
+				"Name": "Async",
+				"Args": {
+					"configure": [
+						{
+						"Name": "Console",
+						"Args": {
+							"restrictedToMinimumLevel": "Information",
+							"outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} {UserId} {Username} [{Level}] [ReqId: {RequestId}] {Message}{NewLine}{Exception}"
+						}
+						}
+					]
+				}
 			},
+			{
+				//Settings for other sinks
+			}
 
-*	Under ``RollingFile``, change the location of the logfiles by editing the value for ``pathFormat``.
-	For example::
-
-		{
-			"Name": "RollingFile",
-			"Args": { "pathFormat": "c:/logfiles/vonk-{Hour}.log" }
-		},
-
-You can also limit the size of the log file, which is 1GB by default, with the ``fileSizeLimitBytes`` option.
-If you set the value for this argument to ``null``, you remove the limit. Serilog retains a maximum of 31 files.
-This value can also be removed or changed, by providing the ``retainedFileCountLimit`` argument::
+The Console is notoriously slow at processing log statements, so it is recommended to limit the number of statements for this sink. Use the ``restrictedToMinimumLevel`` to do so.
 
-	{
-		"Name": "RollingFile",
-		"Args": { "pathFormat": "c:/logfiles/vonk-{Hour}.log", "retainedFileCountLimit": "24" }
-	},
+Settings for the Console sink:
 
-Seq
-^^^
-For the ``Seq`` sink, you can also specify arguments. One of them is the server URL for your
-Seq server:
+	* ``outputTemplate``: What information will be in each log line. Besides regular text you can use placeholders for information from the log statement:
```

Comment:
Do all these log setting ensure no PHI information is logged?  If not, might want to indicate which might with a warning?

Reply:
Vonk does not log any content from the resources it processes, so no PHI. Except for the userId / userName maybe. But it is a good idea to mention that explicitly.

Reply:
OK, userid is relatively 'obscure' but userName maybe not.
Also thought about the concept of 'data at rest' -- that if any of these logs are plaintext files, anyone with access to the file system can see this information.   One ''solution" could be to make note of this and recommend that the customer use an encrypted file system if that is a requirement (or concern